.buildkite/publishContent.ts

```diff
@@ -54,14 +54,30 @@ async function upsertTemplate(template: Template) {
     );
   }
 
+  const upload = await client.uploads.createFromLocalFile({
+    localPath: `./screenshots/${template.slug}.png`,
+  });
+
   if (records.length) {
     console.log(
       `${chalk.cyanBright(
         `https://buildkite.com/templates/${template.slug}`
       )} ${chalk.whiteBright("→")} ${chalk.cyan(template.title)}`
     );
 
-    return client.items.update(records[0].id, payload);
+    const item = await client.items.update(records[0].id, {
+      ...payload,
+      preview: {
+        upload_id: upload.id,
+      },
+    });
+
+    // Hose the old preview image now that it's no longer being referenced.
+    if (records[0].preview) {
+      await client.uploads.destroy(records[0].preview.upload_id);
```

Comment:
Just had a thought on what happens to references to these in production... I probably can't do this until the site has rebuilt. 

🤔

Reply:
Actually, I think it's OK because the URLs are served from their CDN so they'll be cached.